Tests/fixtures/clienthints-app.yml

```diff
@@ -2233,3 +2233,231 @@
+    name: Telia Trygg
+    version: ""
+  device:
+    type: smartphone
+    brand: Huawei
+    model: P smart Z
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 5.0; SM-N9005 Build/LRX21V; wv) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.74 Mobile Safari/537.36
+  headers:
+    http-x-requested-with: com.fsecure.ms.upc.ch
+  os:
+    name: Android
+    version: "5.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: UPC Internet Security
+    version: ""
+  device:
+    type: phablet
+    brand: Samsung
+    model: Galaxy Note 3
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 12; SM-G975F Build/SP1A.210812.016; wv) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Mobile Safari/537.36
+  headers:
+    http-x-requested-with: com.fsecure.ms.sfr
+  os:
+    name: Android
+    version: "12"
+    platform: ""
+  client:
+    type: mobile app
+    name: SFR Sécurité
+    version: ""
+  device:
+    type: smartphone
+    brand: Samsung
+    model: Galaxy S10+
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; SM-A202F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.6099.43 Mobile Safari/537.36
+  headers:
+    http-x-requested-with: com.fsecure.ms.sonera
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: Telia Turvapaketti
+    version: ""
+  device:
+    type: smartphone
+    brand: Samsung
+    model: Galaxy A20e
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; SM-G965U Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/99.0.4844.88 Mobile Safari/537.36 DaumApps/6.9.27 DaumDevice/mobile
+  headers:
+    http-x-requested-with: net.daum.android.daum
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: Daum
+    version: 6.9.27
+  device:
+    type: smartphone
+    brand: Samsung
+    model: Galaxy S9+
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.4; en-us; Nexus 4 Build/JOP24G) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30
+  headers:
+    http-x-requested-with: com.ezt.vpn
+  os:
+    name: Android
+    version: "4.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: EZVPN
+    version: ""
+  device:
+    type: smartphone
+    brand: Google
+    model: Nexus 4
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; 5007G Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/88.0.4324.181 Mobile Safari/537.36
+  headers:
+    http-x-requested-with: com.nocardteam.nocardvpn
+  os:
+    name: Android
+    version: "10"
+    platform: ""
+  client:
+    type: mobile app
+    name: NoCard VPN
```

Comment:
Current name is `NoCard Lite` https://play.google.com/store/apps/details?id=com.nocardteam.nocardvpn.lite&hl=en_US

Reply:
oh, I did notice that the `lite` was added to the ID (even though I googled `"com.nocardteam.nocardvpn"`). But, yes, `NoCard VPN Lite` sounds good.
Would it not make sense to split VPNs into separate category? Although that would also mean that we need to split podcast apps, and probably some other types, too.